runtime/pdo/mysql/mysql_pdo_driver.cpp

```diff
@@ -0,0 +1,100 @@
+// Compiler for PHP (aka KPHP)
+// Copyright (c) 2021 LLC «V Kontakte»
+// Distributed under the GPL v3 License, see LICENSE.notice.txt
+
+#include <memory>
+#include <mysql/mysql.h>
+
+#include "runtime/array_functions.h"
+#include "runtime/pdo/mysql/mysql_pdo_driver.h"
+#include "runtime/pdo/mysql/mysql_pdo_emulated_statement.h"
+#include "runtime/pdo/pdo.h"
+#include "runtime/pdo/pdo_statement.h"
+#include "server/database-drivers/mysql/mysql.h"
+#include "server/database-drivers/mysql/mysql-connector.h"
+#include "server/database-drivers/mysql/mysql-response.h"
+#include "server/database-drivers/adaptor.h"
+#include "server/php-queries.h"
+
+namespace pdo::mysql {
```

Comment:
Cool idea about using namespaces, what about to use it in pdo.h/cpp and pdo_statement.h/cpp too for consistency?